bfdd/bfdd_vty.c

```diff
@@ -878,9 +878,12 @@ DEFUN_NOSH(show_debugging_bfd,
 	   "BFD daemon\n")
 {
 	vty_out(vty, "BFD debugging status:\n");
-	vty_out(vty, "  Peer events debugging.\n");
-	vty_out(vty, "  Zebra events debugging.\n");
-	vty_out(vty, "  Network layer debugging.\n");
+	if (bglobal.debug_peer_event)
+		vty_out(vty, "  Peer events debugging is on.\n");
```

Comment:
nit: maybe showing that's explicitly off would be handy as well?

Reply:
no. it is not done for other protocols.

Reply:
👌